Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java

```diff
@@ -153,7 +132,6 @@ protected void resetTest(String testName) throws Exception {
         if (!currentTestName.equals(testName)) {
             return;
         }
-        wireMock=null;
```

Comment:
I have test on my local machine , everything works good. The mapping should be reset when a new wiremock instance is generated.